redfish-core/lib/update_service.hpp

```diff
@@ -764,6 +763,25 @@ inline void
     }
 }
 
+/**
+ * UpdateServiceActionsOemConcurrentUpdate class supports handle POST method for
+ * concurrent update action.
+ */
+inline void requestRoutesUpdateServiceActionsOemConcurrentUpdate(App& app)
+{
+    BMCWEB_ROUTE(app, "/redfish/v1/UpdateService/Actions/Oem/"
+                      "OemUpdateService.ConcurrentUpdate/")
+        .privileges(redfish::privileges::postUpdateService)
+        .methods(boost::beast::http::verb::post)(std::bind_front(
+            [&app](App&, const crow::Request& req,
+                   const std::shared_ptr<bmcweb::AsyncResp>& asyncResp) {
+        handleUpdateServicePost(app, req, asyncResp,
+                                "/redfish/v1/UpdateService/Actions/Oem/"
+                                "OemUpdateService.ConcurrentUpdate");
```

Comment:
Same here, put the URL all in one string.
[https://github.com/openbmc/bmcweb/blob/master/COMMON_ERRORS.md#10-urls-constructed-in-aggregate](https://github.com/openbmc/bmcweb/blob/master/COMMON_ERRORS.md#10-urls-constructed-in-aggregate)

Reply:
Done